### ISSUE TITLE
mapa na fedrowaniu pokazująca instytucje i znaczniki odpowiadające określonych cechom

### ISSUE THREAD

**Issue by AgnieszkaZdanowicz** [watchdogpolska/feder#937]
W zagadnieniu #628 pojawił się pomysł, żeby na fedrowaniu stworzyć mapę, która będzie pokazywała np. wszystkie instytucje z monitoringu i to, czy odpowiedziały czy nie. Dla monitoringu Czy urzędy gmin przekazują bez podstawy prawnej listy wyborców? robiliśmy taką mapę na google: https://www.google.com/maps/d/viewer?mid=1w2bEmgu9kzsIQ3VIL7fvtXIgU3MTq35P&shorturl=1&ll=52.105196363050524%2C19.11869215000002&z=6 i wolontariuszka ręcznie te dane wrzucała. To by nam niesamowicie ułatwiło pracę, gdyby takie mapy były generowane przez fedrowanie

**Comment by xmichaelx** [comment]
Hej, jeżeli potrzebujecie dalej takiej wizualizacji to mógłbym w Leaflet.js dorzucić robienie czegoś takiego na podstawie generowanych outputów i dodać do istniejacych intefejsów.

**Comment by ad-m** [comment]
@AgnieszkaZdanowicz @dzejkobi , co sądzicie, aby teraz to zorientować wokół nowododanych znaczników? 

@xmichaelx , czy masz na myśli przez istniejące interfejsy także stronę główną? Jak będzie to działać na stronie głównej? Znasz się na front-endzie, czy back-endzie, czy oba? Musielibyśmy geokodować dane instytucji.

**Comment by xmichaelx** [comment]
@ad-m Oba. Jeżeli macie zadania, które waszym zdaniem są bardziej pilne to zacznę od nich. Praca i synchronizacja z mojej strony raczej wieczorami i weekendami.

**Comment by AgnieszkaZdanowicz** [comment]
@ad-m moim zdaniem to by było najlepsze rozwiązanie, gdybyśmy mogli wyświetlać gminy na mapie wg tagów, które nadajemy + wg znacznika "otrzymano odpowiedź", który zaznacza się automatycznie

@xmichaelx to super, że chcesz nam pomóc, bo wykorzystujemy fedrowanie coraz częściej i rozwój nowych funkcjonalności jest dla nas bardzo ważny. Ja znam produkt tylko od strony biznesowej, a @ad-m i @dzejkobi również od strony programistycznej

**Comment by xmichaelx** [comment]
@AgnieszkaZdanowicz Pani Agnieszko, chętnie wspomogę @ad-m  i @dzejkobi - jeżeli macie Panie/Panowie jakiś kanał gdzie można byłoby się zsynchronizować i jakoś zadaniować, żeby pchnąć robotę tam gdzie uważacie za słuszne.

**Comment by dzejkobi** [comment]
Odnośnie mapy, dobrze byłoby zacząć od zdefiniowania wymagań i wstępnego zaprojektowania rozwiązania, choćby w formie zwięzłego opisu. Pewnie na następnym spotkaniu statusowym moglibyśmy poruszyć ten temat. Myślę, że pozycje na mapie moglibyśmy filtrować na podstawie róznych kryteriów, tagi mogłyby być jednym z nich.

@xmichaelx Dzięki za zainteresowanie projektem. Chętnie skorzystamy z pomocy. Prawie wszystkie bieżące zadania są zdefiniowane tu w issues, a priorytety i planowanie określamy na spotkaniu statusowym, które odbywa się zwykle co 2 tygodnie w środy o 19:00, więc proponuję wzięcie udziału. Następne jest zaplanowane na 21.04 19:00. Poza tym komunikujemy się przez GitHuba, częściowo na Slacku i mailowo.

**Comment by ad-m** [comment]
@xmichaelx , możesz podesłać na adam.dobrawy@siecobywatelska.pl swój e-mail? Dodam cię do kalendarza, abyś wiedział o statusach. Jeżeli nie możesz – daj znać, zorganizujemy to w inny sposób.

**Comment by xmichaelx** [comment]
W porządku - w takim razie zajmę się najpierw kwestią geocodingu dla instytucji.

**Comment by dzemeuksis** [comment]
@PiotrIw , rozmawialiśmy jakiś czas temu odnośnie czegoś takiego, tj. jakiejś automatycznej wizualizacji wyników monitoringu na mapie.

@AgnieszkaZdanowicz , czy nie jest tak, że tego typu funkcjonalności postanowiliśmy rozwijać raczej w Fajnych Danych a na Fedrowaniu już w to nie brnąć?

**Comment by AgnieszkaZdanowicz** [comment]
@dzemeuksis tak, dokładnie, moim zdaniem to będzie realizowane w wysłanych danych i tutaj chyba nie ma po co rozwijać takiego samego mechanizmu. Chociaż nie wiem dokładnie, jak mają działać Fajne Dane - tak będziemy mogli dosyć łatwo wrzucić dane uzyskane przez fedrowanie?

**Comment by dzemeuksis** [comment]
@AgnieszkaZdanowicz , na ile łatwo będziemy mogli wrzucać dane z Fedrowania do FD, to wyjdzie w praniu. Lada moment będziemy to testować na przykładzie "Wydanych danych".

Przy okazji, czy to zgłoszenie nie jest w jakimś stopniu duplikatem tego: #327 ? Jaki jest związek między tymi zgłoszeniami?
